app/src/main/java/com/courseproject/tindar/usecases/likelist/LikeListInteractor.java

```diff
@@ -0,0 +1,28 @@
+package com.courseproject.tindar.usecases.likelist;
+
+public class LikeListInteractor implements LikeListInputBoundary {
+    /** This class checks if other is in userId likeList. If not, userId is added to other likeList.
```

Comment:
Might be worth explaining the meta reasons why this class exists. Honestly every confused about why this exists, because the only thing this class currently does is hold a likeListDsGateway object, call that object's methods, and do logic on it.

Reply:
Dsgateway object holds user information, it is compiled from database and sent to likelistintrractor as a single object rather than separate strings or however the database stores info

Reply:
Similar to (https://github.com/CSC207-2023Y-UofT/course-project-tindar/pull/26/files#r1280577368). Elaborate and add to javadoc.

Reply:
Similar to (https://github.com/CSC207-2023Y-UofT/course-project-tindar/pull/26/files#r1280577368). Elaborate and add to javadoc.